syft/workers/base.py

```diff
@@ -510,9 +529,9 @@ def send_command(
         name, target, args_, kwargs_ = message
 
         try:
-            ret_val = self.send_msg(
-                CommandMessage(name, target, args_, kwargs_, return_ids), location=recipient
-            )
+            action = ComputationAction(name, target, args_, kwargs_, return_ids)
+            message = CommandMessage(action)
+            ret_val = self.send_msg(message, location=recipient)
```

Comment:
I think this is fine, but I still wonder about adding static pseudo-constructors like `CommandMessage.computation()` and `CommandMessage.communication()` to make snippets like this a bit more terse.

Reply:
(Maybe YAGNI applies here though.)

Reply:
Haha, I was just going to do that. I do think it would be clearer to have 
```
message = CommandMessage.computation(name, target, args_, kwargs_, return_ids)
```
instead of the 2 lines above.
Do you think we should refrain from implementing that until there is a more urgent need to do so?

Reply:
Nah, if you're onboard, let's go for it. It reads nicely and leaves the calling code cleaner. 👍